### PR TITLE
Add Propel Data

### PR DESCRIPTION
**Pull request by ykdojo**
Context: https://www.snowflake.com/blog/startup-spotlight-api-on-top-of-snowflake-propel/

### CODE REVIEW

File: Tools/Propel.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Propel Advantages
+
+- A managed serverless service
+- Low latency, typically in milliseconds
+- Built-in timezone handling
+- Semantic layer for data modeling with GraphQL APIs
+
+## Propel Disadvantages
+
+- Learning curve due to the novel nature of the service
```
Comment (acossta) [inline Tools/Propel.md:21]:
```suggestion
## Use cases

- Embedded analytics: build analytics products.
- Data APIs: Instant low-latency APIs over any data source.
- Data Sharing: Give your customers their data.
- Self-service custom reporting: Empower your customers to build their own reports.
- ```

File: Tools/Propel.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## Propel Official Documentation
+
+https://www.propeldata.com/docs
+
+## Propel Advantages
+
+- A managed serverless service
+- Low latency, typically in milliseconds
+- Built-in timezone handling
+- Semantic layer for data modeling with GraphQL APIs
```
Comment (acossta) [inline Tools/Propel.md:17]:
```suggestion
- A managed serverless service
- Ingest data and speed up queries from any data source.  Low latency, typically in milliseconds
- Build at any layer of the stack: SQL, APIs, or React UI Components.
- Semantic layer for data modeling with GraphQL APIs
```

**Review by acossta — APPROVED** [review]

File: Tools/Propel.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Embedded analytics: build analytics products.
+- Data APIs: Instant low-latency APIs over any data source.
+- Data Sharing: Give your customers their data.
+- Self-service custom reporting: Empower your customers to build their own reports.
+
+%% wiki footer: Please don't edit anything below this line %%
+
+## This note in GitHub
+
+<span class="git-footer">[Edit In GitHub](https://github.dev/data-engineering-community/data-engineering-wiki/blob/main/Tools/Databricks.md "git-hub-edit-note") | [Copy this note](https://raw.githubusercontent.com/data-engineering-community/data-engineering-wiki/main/Tools/Databricks.md "git-hub-copy-note")</span>
```
Comment (JPHaus) [inline Tools/Propel.md:31]:
```suggestion
<span class="git-footer">[Edit In GitHub](https://github.dev/data-engineering-community/data-engineering-wiki/blob/main/Tools/Propel.md "git-hub-edit-note") | [Copy this note](https://raw.githubusercontent.com/data-engineering-community/data-engineering-wiki/main/Tools/Propel.md "git-hub-copy-note")</span>
```

File: Tools/Propel.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+<span class="git-footer">Was this page helpful?
+[👍](https://tally.so/r/mOaxjk?rating=Yes&url=https://dataengineering.wiki/Tools/Databricks) or [👎](https://tally.so/r/mOaxjk?rating=No&url=https://dataengineering.wiki/Tools/Databricks)</span>
```
Comment (JPHaus) [inline Tools/Propel.md:34]:
```suggestion
[👍](https://tally.so/r/mOaxjk?rating=Yes&url=https://dataengineering.wiki/Tools/Propel) or [👎](https://tally.so/r/mOaxjk?rating=No&url=https://dataengineering.wiki/Tools/Propel)</span>
```

**Review by JPHaus — APPROVED** [review]
Thanks for sharing this tool!

**Comment by ykdojo** [comment]
Thank you @JPHaus. It seems like it's not visible on the site yet. Will it need to be deployed?

**Pull request merged**